test/gw-ci/CMakeLists.txt

```diff
@@ -99,40 +202,6 @@ if (WORKFLOW_TESTS)
     # "gdas_marineanlchkpt"
     # "gdas_marineanlfinal"
     )
-  add_cycling_tests(${pslot} ${YAML_PATH} ${HOMEgfs} ${RUNTESTS} ${PROJECT_SOURCE_DIR} "${TASK_LIST}")
-  set(letkf FALSE)
-endif()
-
-option(RUN_GW_CI "Enable the global-workflow CI tests" OFF)
-if (RUN_GW_CI)
-  # Aero-Land DA, C96
-  # -----------------
-  set(pslot "Aero-Snow-3DVAR-C96")
-  set(YAML_PATH ${HOMEgfs}/ci/cases/pr/C96C48_hybatmaerosnowDA.yaml)
-  set(TASK_LIST)  # empty list for now
-  add_cycling_tests(${pslot} ${YAML_PATH} ${HOMEgfs} ${RUNTESTS} ${PROJECT_SOURCE_DIR} "${TASK_LIST}")
-
-  # Atm DA, C96/C48
-  # ---------------
-  set(pslot "Atm-hyb-C96C48")
-  set(YAML_PATH ${HOMEgfs}/ci/cases/pr/C96C48_ufs_hybatmDA.yaml)
-  set(TASK_LIST)  # empty list for now
-  add_cycling_tests(${pslot} ${YAML_PATH} ${HOMEgfs} ${RUNTESTS} ${PROJECT_SOURCE_DIR} "${TASK_LIST}")
-
-  # GFSv17, 3DVAR prototype
```

Comment:
Maybe I missed it, but the gfsv17 prototype test doesn't seem to be in.
Add it back and wrap it under a specific flag so we don't run it for every pr.

Reply:
Ah yes, OK, will do

Reply:
@guillaumevernieres OK, I put it back under the flag ```TEST_GFS17```